### PR TITLE
(fix) Schedule Cards

### PR DESCRIPTION
**Pull request by mazenchami**
# Description

[Trello Card](160-welcome-card) — #128 
[Trello Card](162-trivia-card) — #130 
[Trello Card](166-after-party-is-may-18-dont-include-the-19th) — #132 
[Trello Card](167-workshops-disclaimer) — #138 

Several changes here:
1. The `Welcome` card now better highlights our emcee, Ken LaFrance
2. The `Trivia Card` is now a thing (`variant` and all)
3. Fixed the After Party date on the `Venues` tab
4. Added a Workshops disclaimer on the `Wed` tab on the `Schedule` tab

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
| <br/><br/><br/> | <br/><br/><br/> |

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

File: app/screens/ScheduleScreen/ScheduleScreen.tsx [hunk 1]
```diff
@@ -204,43 +206,50 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
...
-                  index < scheduleIndex ||
-                  (index === scheduleIndex && itemIndex < eventIndex) ||
-                  isConfOver
-                }
+      <>
+        {index === 0 && (
+          <View style={$workshopBanner}>
+            <Text style={$workshopBannerText}>{translate("scheduleScreen.workshopBanner")}</Text>
+          </View>
+        )}
```
Comment (mazenchami) [inline app/screens/ScheduleScreen/ScheduleScreen.tsx:214]: this is the only update here, the rest is just tabbing over

File: app/services/api/webflow-helpers.ts [hunk 2]
```diff
@@ -33,16 +33,24 @@ export const cleanedSchedule = ({
...
+        "recurring-event": recurringEvents?.find(({ _id }) => _id === schedule["recurring-event"]),
+        "speaker-2": speakers?.find(({ _id }) => _id === schedule["speaker-2"]),
+        "speaker-3": speakers?.find(({ _id }) => _id === schedule["speaker-3"]),
+        "speaker-2-2": speakers?.find(({ _id }) => _id === schedule["speaker-2-2"]),
+        "speaker-3-2": speakers?.find(({ _id }) => _id === schedule["speaker-3-2"]),
+        day: WEBFLOW_MAP.scheduleDay[schedule.day] ?? WEBFLOW_MAP.scheduleDay["2e399bc3"],
+        talk: talks?.find((talk) => talk._id === schedule["talk-2"]),
+        type: isTriviaShow
+          ? WEBFLOW_MAP.triviaShow.title
+          : WEBFLOW_MAP.scheduleType[schedule["event-type"]],
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/services/api/webflow-helpers.ts:50]: Chatted with @mazenchami this seemed like more work on the webflow side.

File: app/services/api/webflow-helpers.ts [hunk 3]
```diff
@@ -33,16 +33,24 @@ export const cleanedSchedule = ({
...
-      location: WEBFLOW_MAP.location[schedule.location],
-      "recurring-event": recurringEvents?.find(({ _id }) => _id === schedule["recurring-event"]),
-      "speaker-2": speakers?.find(({ _id }) => _id === schedule["speaker-2"]),
-      day: WEBFLOW_MAP.scheduleDay[schedule.day] ?? WEBFLOW_MAP.scheduleDay["2e399bc3"],
-      talk: talks?.find((talk) => talk._id === schedule["talk-2"]),
-      type: WEBFLOW_MAP.scheduleType[schedule["event-type"]],
-      workshop: workshops?.find(({ _id }) => _id === schedule.workshop),
-    }))
+    .map((schedule) => {
+      const isTriviaShow = schedule["event-title"] === WEBFLOW_MAP.triviaShow.title
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/services/api/webflow-helpers.ts:37]: @mazenchami said this was more work for the webflow side so conditionals it is.

**Review by morganick — COMMENTED** [review]
Just one comment.

Otherwise, looks good and works as expected.

File: app/screens/VenuesScreen/VenuesScreen.tsx [hunk 4]
```diff
@@ -46,7 +46,11 @@ const useVenuesSections = (): {
             imageSource: venue["venue-image-s"].map((image) => ({ uri: image.url })),
             title: venue.name,
             subtitle: `${WEBFLOW_MAP.venueTag[venue.tag]} • ${
-              WEBFLOW_MAP.venueTag[venue.tag] === "Workshop" ? "May 17" : "May 18-19"
+              WEBFLOW_MAP.venueTag[venue.tag] === "Workshop"
+                ? "May 17"
+                : WEBFLOW_MAP.venueTag[venue.tag] === "After Party"
+                ? "May 18"
+                : "May 18-19"
```
[the remark this answers is not in the document]
Comment (mazenchami) [reply app/screens/VenuesScreen/VenuesScreen.tsx]: extracted into a switch statement: https://github.com/infinitered/ChainReactApp2023/pull/144/commits/f968d4d5ce26cf20adff6dd4a5d48b9066784a4b

File: app/screens/VenuesScreen/VenuesScreen.tsx [hunk 5]
```diff
@@ -45,13 +56,7 @@ const useVenuesSections = (): {
...
             imageSource: venue["venue-image-s"].map((image) => ({ uri: image.url })),
             title: venue.name,
-            subtitle: `${WEBFLOW_MAP.venueTag[venue.tag]} • ${
-              WEBFLOW_MAP.venueTag[venue.tag] === "Workshop"
-                ? "May 17"
-                : WEBFLOW_MAP.venueTag[venue.tag] === "After Party"
-                ? "May 18"
-                : "May 18-19"
-            }`,
+            subtitle: `${WEBFLOW_MAP.venueTag[venue.tag]} • ${getVenueDate(venue.tag)}`,
```
Comment (morganick) [inline app/screens/VenuesScreen/VenuesScreen.tsx:59]: 🙌

**Review by morganick — APPROVED** [review]
👍 LGTM nice work @mazenchami

**Pull request merged**